docs/getting-started/installation/wpilib.md

```diff
@@ -0,0 +1,6 @@
+# Installing WPILib
+Luckily the installation process for WPILib isn't changed much (if at all) by the use of Kotlin in your projects.
+
+To install WPILib you can simply follow the guide contained on the WPILib wiki, with a few special instructions. You can find this guide [HERE](https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-2/wpilib-setup.html). Once you reach the page where it asks which install mode you'd like, make sure to select "Everything", and then on the next page select "Skip and don't use VS Code".
```

Comment:
avoid simply

Link the guide inline with the word 'guide', maybe?

I actually would recommend they keep VSCode because it's the simplest way to update vendor deps. It doesn't seem like the FRC plugin does that

Reply:
On our team we've always just used the gradle task, vendor deps automatically update then if you build on the network because its a glorified wget to pull in the vendordep json

Reply:
I don't think that's true. If I set the version in `Phoenix6.json` to `24.2.0`, then run `./gradlew build`, it won't download the new json and replace it. But if I then go into vscode and check for vendorlib updates, it recognizes there's an update and downloads the 'new' json file